src/test/java/edu/unc/lib/boxc/migration/cdm/MigrationTypeReportCommandIT.java

```diff
@@ -0,0 +1,34 @@
+package edu.unc.lib.boxc.migration.cdm;
+
+import edu.unc.lib.boxc.migration.cdm.services.MigrationTypeReportService;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * @author krwong
+ */
+public class MigrationTypeReportCommandIT extends AbstractCommandIT {
+    private MigrationTypeReportService typeReportService;
+
+    @Before
+    public void setup() throws Exception {
+        typeReportService = new MigrationTypeReportService();
+    }
+
+    @Test
+    public void generateMigrationTypeReportTest() throws Exception {
```

Comment:
It'd be good to add a test that verifies what happens if there is no post migration report. I'm guessing it'll be a FileNotFound exception currently, which might be fine.